cypress/integration/login.js

```diff
@@ -21,4 +21,11 @@ describe('Login functionality', function() {
         cy.get('.user-info__title').contains('TestCon User 1')
         cy.get('.main-nav').find('li').should('have.length', 2)
     })
+
+    it('Should select the current date', function () {
+        var d = new Date();
+        var currentDay = d.getDate();
+        cy.get('.calendar__date').contains(currentDay).click({force:true})
```

Comment:
You should not click on the day. The task is to validate what date is selected by default.

Reply:
fixed